sdks/python/apache_beam/io/gcp/dicomio.py

```diff
@@ -248,31 +268,31 @@ def __init__(self, destination_dict, credential=None):
     self.destination_dict = destination_dict
 
   def expand(self, pcoll):
-    return pcoll | beam.ParDo(StoreInstanceBytes(self.destination_dict, self.credential))
+    return pcoll | beam.ParDo(
+        StoreInstanceBytes(self.destination_dict, self.credential))
 
 
 class StoreInstanceBytes(beam.DoFn):
   """A DoFn execute every file input."""
```

Comment:
Can you please add more descriptive comments for classes and methods.

Reply:
Examples and more comments added